webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/jdbc/HistorizationJdbcRepository.scala

```diff
@@ -244,7 +244,8 @@ class HistorizationJdbcRepository(db: Doobie) extends HistorizationRepository wi
         _  <- Update[DB.SerializedRuleDirectives]("""
                  insert into rulesdirectivesjoin (rulepkeyid, directiveid)
                  values (?, ?)
-               """).updateMany(r.directiveIds.toList.map(d => DB.SerializedRuleDirectives(pk, d.value)))
+               """).updateMany(r.directiveIds.filter(_.revId.isEmpty).toList.map(d => DB.SerializedRuleDirectives(pk, d.id.value)))
```

Comment:
I don't understand why we do not serialize with revision id here ? 

It means we don't store the value if there is a revId, and we only keep track of directive with no revision (head)

Reply:
We should be able to totally remove historization repository with that change, since everything will be in git. So I didn't want to loose time on making it correct. 
If we are force to keep it for an unforseen reason, we will see what we historize exactly.